docs/sources/panels-visualizations/visualizations/stat/index.md

```diff
@@ -39,15 +39,15 @@ The panel automatically adjusts the layout depending on available width and heig
 
 ## Value options
 
-Use the following options to refine how your visualization displays the value:
+Use the following options to refine how your visualization displays its value/s:
 
 ### Show
 
-Choose how Grafana displays your data.
+Displays a single value per column or series, or shows values for each row.
```

Comment:
to be more consistent with how the other options are described at the high level: as an action you, the user, can take vs what action the system is taking
```suggestion
Display a single value per column or series, or show values for each row.
```